index.html

```diff
@@ -3,8 +3,38 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <h1 class="header">
+            <p class="logo">My Logo</p>
+            <div><img class="menu" src="Union.png" alt="MenuBar"></div>
+        </h1>
```

Comment:
The font and color that is being used for the logo doesn't seem to match the one on figma. Also the margin around the navigation can be reduced so that it doesn't appear as it is right now. Addressing these minor issues will help to make the UI more attractive to recruiters during job search.

YOUR NAV
![nononoNav](https://user-images.githubusercontent.com/84930355/227286830-3449af26-256f-4206-bd69-d40d3dffe2da.PNG)


FIGMA NAV
![oodNav](https://user-images.githubusercontent.com/84930355/227286938-754fba9e-b811-4988-bc3b-bdc206841249.PNG)